openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/TestSuiteRepository.java

```diff
@@ -460,6 +461,24 @@ public RestUtil.DeleteResponse<TestSuite> deleteLogicalTestSuite(
     return new RestUtil.DeleteResponse<>(updated, changeType);
   }
 
+  /**
+   * Always delete as if it was marked recursive. Deleting a Logical Suite should
+   * just go ahead and clean the Ingestion Pipelines
+   */
+  private void deleteChildIngestionPipelines(UUID id, boolean hardDelete, String updatedBy) {
+    List<CollectionDAO.EntityRelationshipRecord> childrenRecords =
+        daoCollection
+            .relationshipDAO()
+            .findTo(id, entityType, Relationship.CONTAINS.ordinal(), Entity.INGESTION_PIPELINE);
+
+    if (childrenRecords.isEmpty()) {
+      LOG.info("No children to delete");
```

Comment:
this should be debug

Reply:
updated in the entityRepo as well, thanks